### DISCUSSION TITLE
Is it possible to create types from dependencies?

### DISCUSSION THREAD

**Discussion by n1c** [spatie/typescript-transformer#86] [Q&A]
I have a project that includes some shared code as a dependency. I'd like to have the types from that code also generate types (it has the correct annotations).

I tried including the path to the library in `auto_discover_types` with no lucky.

Any clues? Thanks in advance!

**Comment by rubenvanassche** [comment]
Normally this should work, could you send me a test repository showing this problem?

**Reply by n1c** [reply]
following up here (sorry it took ages). I made 2 small repos and it does indeed work as expected:

https://github.com/n1c/test-types-exporter/tree/export-types
https://github.com/n1c/test-types-lib

I'll have to dig deeper into where my specific was going wrong to see if it's maybe just user error on my side.
